integrations/hello-world-ng5/package.json

```diff
@@ -50,6 +42,6 @@
     "protractor": "5.1.2",
     "ts-node": "4.1.0",
     "tslint": "5.9.1",
-    "typescript": "2.7.1"
+    "typescript": "2.8.1"
```

Comment:
What was the reason for this version bump?
I had it working with 2.7.1

Reply:
https://github.com/ReactiveX/rxjs/issues/4512#issuecomment-464892499

I did the same thing, copied the directories, it did not work, I did it through the paths, but it did not work there